minecraft_servers/minevale/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "server_name": "minevale",
+  "nice_name": "MineVale",
+  "direct_ip": "minevale.de",
+  "server_wildcards": [
+    "%.minevale.de",
+    "%.minevale.eu"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://minevale.de",
```

Comment:
1. https not supported
2. minevale = opsucht? Why the redirect?

Reply:
Fixed